.env.example

```
@@ -113,6 +113,44 @@ BREVO_LIST_ID=
 NEXT_PUBLIC_BREVO_CONVERSATIONS_ID=
 NEXT_PUBLIC_BREVO_CONVERSATIONS_GROUP_ID=
+# =============================================================================
+# Phone Verification Services
+# =============================================================================
+
+# KudiSMS (for African phone numbers)
+# Get from: KudiSMS Dashboard → Settings → API Keys
+KUDISMS_API_KEY=your_kudisms_api_key
+KUDISMS_APP_NAME_CODE=your_app_name_code
+KUDISMS_TEMPLATE_CODE=your_template_code
+KUDISMS_SENDER_ID=Noblocks
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KUDISMS_SENDER_ID key should go before the KUDISMS_TEMPLATE_CODE key

(UnorderedKey)